server/services/v1/database/search_indexer.go

```diff
@@ -97,6 +97,10 @@ func (i *SearchIndexer) OnPostCommit(ctx context.Context, tenant *metadata.Tenan
 				return err
 			}
 
+			if tableData == nil {
```

Comment:
From what I can see, when we add an index row, on the post commit those updates are being passed here. All the index key values have `nil` for the tableData since they only adding keys. That is causing an issue here.

Reply:
ah, now I see - makes sense.